docs/reference/organizations-on-nuget-org.md

```diff
@@ -0,0 +1,105 @@
+
+### Transform existing account to an organization
+
+If you're managing packages as a team using a single user account and would like to convert that account into an organization, use the **Transform your account to an organization** option on the **Manage Organizations** page:
+
+![Option on nuget.org to transform an existing account to an organization](media/org-transform-option.png)
+
+On the next page, specify different user account to assign as the administrator of the organization, then select **Transform**.
+
+![Entering information for transforming a user account to an organization](media/org-transform-page.png)
+
+> [!Warning]
+> Account conversion is irreversible: you cannot transform an organization back to a user account.
+
+## Managing organization members
+
+As the organization administrator, you can add members by providing each member's nuget.org *user account name*; email addresses cannot be used. You then mark each member as a collaborator or administrator with the following permissions:
+
+| Permission | Collaborator | Administrator |
+| --- | --- | --- |
+| Manage the organization's packages<br/>(submit new packages, update or unlist existing packages) | Yes | Yes |
+| Change organization metadata<br/>(email address, notification settings) | No | Yes |
+| Manage organization members | No | Yes |
+| Request or act on co-ownership requests for organization packages | No | Yes |
+
+## Managing packages
+
+As an organization member, you can see all the organization's packages in the **Packages** section of the organization's details page. On the **Manage Packages** page you can also view packages across your account and all organizations of which you're a member. You can select the account filter to view packages by your account or any specific organization.
+
+![Managing packages with the account filter](media/org-manage-packages-option.png)
+
+## Publishing packages
+
+You publish packages to an organization like you publish packages to a user account, by directly uploading the package to nuget.org or by pushing the package through the `nuget push` or `dotnet nuget push` CLI commands.
+
+### Uploading packages
+
+When you directly upload a new package on the [nuget.org Upload](https://www.nuget.org/packages/manage/upload) page, you assign the package owner to a user or organization account :
+
+![Upload package with account option](media/org-upload-option.png)
+
+When submitting updates to an existing package, you don't see the account dropdown because package ownership is already known to nuget.org.
+
+### Using API keys
+
+To push a package through the `nuget push` or `dotnet nuget push` CLI commands, you must obtain the API key needed by those commands. For details, see [Publish a package](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio#publish-the-package).
```

Comment:
The focus of this section should be to explain that pushing an organization package through the CLI is identical to pushing a user package through the CLI but with an API key scoped to the organization. I don't think the way this is written currently is very clear about that.

```
Pushing an organization package through the `nuget push` or `dotnet nuget push` CLI commands is identical to pushing a user package but with a different API key. To push an organization package, you must use an API key scoped to your organization.

An API key scoped to your organization can be created by selecting the organization in the **Package Owner** drop-down:

![API key with account option](media/org-apikey-option.png)

Any API key you create is applicable only to the chosen user or organization and its packages. Existing API keys or API keys that were created without changing the drop-down are scoped to your user account.

For details on how to push a package using the CLI, see [Publish a package](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio#publish-the-package).
```

Also `obtain the API key` -> `obtain an API key`